sdk/lib/convert/html_escape.dart

```diff
@@ -38,6 +38,23 @@ const HtmlEscape htmlEscape = HtmlEscape();
 ///
 /// Custom escape modes can be created using the [HtmlEscapeMode.HtmlEscapeMode]
 /// constructor.
+///
+/// Example:
+/// ```dart
+/// const sample = 'Text & subject, <, >, "quoted", \'single-quoted\', /';
+///
+/// const htmlEscapeMode = HtmlEscapeMode(
+///     name: 'custom',
+///     escapeLtGt: true,
+///     escapeQuot: false,
+///     escapeApos: false,
+///     escapeSlash: false);
+///
+/// const HtmlEscape htmlEscape = HtmlEscape(htmlEscapeMode);
+/// final htmlEscaped = htmlEscape.convert(sample);
+/// print(htmlEscaped);
+/// // Text &amp; subject, &lt;, &gt;, "quoted", 'single-quoted', /
```

Comment:
`& ` turns in to `&amp` by default and `\`as well? 

Might be confusing that `escapeLtGt = true` being only true changes `&` and `\` when it should escape only 

```
escapeLtGt → bool
Whether to escape '<' and '>'.
```

Reply:
"All modes require escaping of & (ampersand) characters, and may enable escaping of more characters."

Reply:
But you have not specified escaping of &, so it is escaped by default?

What about `\`?

Reply:
**&** escaped by default ( just like in normal constructor )..